monkey/monkey_island/cc/services/agent_plugin_service/i_agent_plugin_service.py

```diff
@@ -54,5 +54,6 @@ def install_agent_plugin_archive(self, agent_plugin_archive: bytes):
         :raises RemovalError: If an error occus while attempting to uninstall a previous
         version of the plugin
         :raises StorageError: If an error occurs while attempting to install the plugin
+        :raises PluginInstallError: If an error occurs while attempting to parse the plugin archive
```

Comment:
Callers don't need to know this much technical detail. The error itself can contain more specifics. Plus, we may decide to raise this error for new reasons, which would make this docstring inaccurate.

```suggestion
        :raises PluginInstallError: If an error occurs while attempting to install the plugin archive
```